lib/licenses.nix

```diff
@@ -498,6 +498,12 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     fullName  = "University of Illinois/NCSA Open Source License";
   };
 
+  nomachine6 = {
```

Comment:
It's not worth adding it to the global licenses list since it won't be used by several packages.

Instead you may define the license in-line in the `meta` block of the package definition. [Here is](https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/maxx/default.nix#L82) an example.